staging/src/k8s.io/component-base/zpages/flagz/flagz.go

```diff
@@ -40,8 +38,7 @@ Warning: This endpoint is not meant to be machine parseable, has no formatting c
 )
 
 var (
-	flagzSeparators         = []string{":", ": ", "=", " "}
-	errUnsupportedMediaType = fmt.Errorf("media type not acceptable, must be: text/plain")
+	delimiters = []string{":", ": ", "=", " "}
```

Comment:
this should also be moved to the util package no?

Reply:
Hmm I guess I didnt move this to the util package since I named it httputil - specific for the common code around validating the http requests to the z-pages. Didnt want to make a common util package that had miscellaneous pieces of code all shoved into it.